package.json

```diff
@@ -23,6 +23,8 @@
     "@mdb/flora": "^0.22.4",
     "@mdx-js/react": "1.6.22",
     "airtable": "0.11.4",
+    "axios": "0.27.2",
```

Comment:
not against using it, but do we have to use `axios`? Are there any scenarios you foresee where `fetch()` wouldn't be able to handle it?

Reply:
So I know Axios is a great third party for fetching data but in the context of this project, what are we accomplishing in using Axios over fetch? Not entirely opposed to it but just want to make sure that I understand why Axios is used in this case. I was doing some reading and stumbled on this [article](https://blog.logrocket.com/axios-vs-fetch-best-http-requests/#:~:text=To%20send%20data%2C%20fetch(),stringify%20method)  Nonetheless I leave on to the both of you to choose what's best in our case :)

Reply:
So we need to make web requests server-side, and we're running Node 16, which doesn't have native fetch functionality, so we need some third-party library. I've just always used axios in Node, but I'm not super tied to it.

However, the current release of Node is v18, which does have native fetch functionality. I think it might be worth trying to migrate, but that might be worth creating a new ticket for?